src/bridge/MatrixHandler.ts

```diff
@@ -956,6 +975,11 @@ export class MatrixHandler {
             return;
         }
 
+        const potentialCodeText = trimCodeMessages(text);
```

Comment:
Agreed that this isn't the best function to be doing this on. My wish is that we refactor this function to act as a pipeline of statements that modify a given event into an IRC message in a stateless context. Perhaps a better place would be `IrcAction.fromMatrixAction`, which is intended to be the point where we format a Matrix Event into an IRC action.